### ISSUE TITLE
support NWChem 7 (get vibrational data)

### ISSUE THREAD

**Issue by drew-parsons** [cclib/cclib#1131]
cclib only supports NWChem versions 6.0, 6.1, 6.5 and 6.6.  It would be helpful if cclib could support NWChem 7 (the latest release is 7.0.2).

In particular I'm interesting in extract vibrational data (vibfreqs, vibfconsts, etc).  I have output generated by nwchem task freq, but `ccget -l` only finds
```
$ cclib-get -l water_freq.out 
Attempting to read water_freq.out
cclib can parse the following attributes from water_freq.out:
  aonames
  atombasis
  atomcoords
  atomnos
  charge
  coreelectrons
  gbasis
  metadata
  mult
  natom
  nbasis
  nmo
  scfenergies
  scftargets
  scfvalues
```
cclib doesn't see the vibrational data ("Infra Red Intensities", "Projected Infra Red Intensities")

**Comment by berquist** [comment]
The next PR will need to activate the NWChem 7.0 tests.

**Comment by berquist** [comment]
I see that the current version on Git master is parsing `vibfreqs` and `vibirs` from the projected analysis block as part of #1057.  The main problem comes from geometry optimizations. I made a comment in #1133 about having some other code for vibrational frequencies, but I don't immediately know what that is.

**Comment by berquist** [comment]
Many fixes for 7.0 are in #1188. Parsing normal mode displacements and the mass-weighted Hessian will be in a follow-up PR. NWChem cannot print reduced masses or force constants.

**Comment by mkhorton** [comment]
Thanks for the update on this @berquist! :) I saw #1188 was now merged, does this mean this issue is now closed too?

**Comment by berquist** [comment]
Not quite.  There are a few things going on here:

1. Parsing 7.0 in general no longer errors after #1188.  Features were still missing.
2. #1198 contains parsing for (projected) normal mode displacements (`vibdisps`) and atomic masses (`atommasses` for all calculations).
3. #1199 contains parsing the (projected) mass-weighted Hessian (`hessian`) and atomic masses (`atommasses` in higher precision printed before frequencies).

This constitutes the limit of what we can currently do with NWChem.  If force constants and reduced masses are of interest, then those will have to be calculated from the Hessian.  (I'm interested in ripping code from Psi4 to do this in the future.)  cclib-side, we can't yet make a distinction between the projected and unprojected quantities, but I doubt most people are interested in unprojected quantities.
